package.json

```diff
@@ -120,6 +120,7 @@
     "@testing-library/jest-dom": "^6.6.3",
     "@testing-library/react": "^16.0.1",
     "@testing-library/user-event": "^12.1.10",
+    "@testing-library/dom": "^10.4.0",
```

Comment:
Why was this added? It doesn't appear to be used. Please revert this change.

Reply:
without this module i am getting error: `Module '"@testing-library/react"' has no exported member 'screen'.`